velox/expression/ExprCompiler.cpp

```diff
@@ -23,6 +23,7 @@
 #include "velox/expression/FieldReference.h"
 #include "velox/expression/LambdaExpr.h"
 #include "velox/expression/SimpleFunctionRegistry.h"
+#include "velox/expression/SpecialFormRegistry.h"
```

Comment:
Same here

Reply:
Yes, because it uses isFunctionCallToSpecialFormRegistered

Reply:
Sorry, I missed that function was moved.